index.js

```diff
@@ -1,18 +1,131 @@
+/* This is the main program for Project 1 - Guess the number - 
+  This program will display the number a user is thinking of
+  
+  Code author: Denis Poirier, BCA Sum2019, drpoirier 
+    Date Started:     Thursday Feb 6 2019
+    Date Completed:   in-process
+    Last Date Revised:Sunday Feb 9 2019    
+*/
+
 const readline = require('readline');
 const rl = readline.createInterface(process.stdin, process.stdout);
 
+// hard coding to min & max to begin with ; later these will be changed as user inputs for what they want
+let min=1;    // not inclusive of 0
+let max=100;  // at first limit the max to 100, then increase it to 500 or give user a choice 
+let cpuGuessNum = 0; // setting to zero as a default ? (also makes it a global so it can change thruought)
+let range = ((max - min)+1); // range = ((max# - min)+1) so it's not 0 
+
+// function source = MDN - The maximum (max) is inclusive and the minimum (min) is inclusive -
+//   This function calculates the best/shortest number of tries
+let maxTries = (Math.round((Math.log2((max)+1))));
+
+function getRandomIntInclusive(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
 function ask(questionText) {
   return new Promise((resolve, reject) => {
     rl.question(questionText, resolve);
   });
 }
 
 start();
-
+  
 async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
+  console.log("Let's play a game where you (human user) make up a number and I (computer program) try to guess it.")
+  console.log("Please think of a number between " + min + " and " + max + " (inclusive), in this " + range + " number range.");
+  let secretNumber = await ask("What is your secret number?\n(This Javascript (JS) program (pgm) won't peek...)\n");
+  console.log('FYI, you entered: ' + secretNumber + ' (Again, this will be kept secret until the end!)'); 
+
+if ((secretNumber >= min) && (secretNumber <= max)) {   
+
+// mainGuessing 
+  cpuGuessNum = getRandomIntInclusive(min, max);   // Guessing a random number in the range to start
+  console.log('(This program currently randomly guesses only the first try) Is it... ' + cpuGuessNum); 
+  let pgmTries = 1; 
+  let userRespYorN = await ask('? (Please type a Y or a N only) -> ');
+    let userRespYorNUC = userRespYorN.toUpperCase().trim();  // to capitalize letters and also trim spaces
+    
+  if (userRespYorNUC === 'Y') {
+    console.log('You stated that this was the correct number, Yea!');  
+    console.log('The program has now tried: ' + pgmTries + ' times.'); // game exits since it's done
+    process.exit();
+  } else if (userRespYorNUC === 'N') {
```

Comment:
Don't forget about our [logical operators](https://bootcamp.burlingtoncodeacademy.com/lessons/javascript/common-operators#anchor/logical_operators), you can always define that something _**!==**_ something else.